.github/workflows/create-tag.yml

```diff
@@ -169,12 +169,6 @@ jobs:
             const annotatedTag = process.env.ANNOTATED_TAG
             const message = process.env.MESSAGE
 
-            // TODO(#215): Once prevent self-review is supported by GitHub, we can remove this condition.
-            if (branch != '${{ github.event.repository.default_branch }}') {
-              core.setFailed(`branch (${branch}) is not the repository default branch (${{ github.event.repository.default_branch }}).`)
-              return
-            }
-
```

Comment:
I would still like this to spit out a warning for reviewers.

Reply:
done, thanks!